components/specification/publish

```diff
@@ -153,18 +153,23 @@ contains() {
 }
 
 devmarker="dev"
-root=published
-
+root=${1:-published}
+root=${root%/}
 
 # Clean up
 
-rm -rf "$root"
+rm -rf "$root/*"
```

Comment:
I will update the description to create an intermediate repo when modifying the Git files.